addons/popochiu/engine/objects/graphic_interface/templates/sierra/sierra_gi.gd

```diff
@@ -0,0 +1,117 @@
+extends PopochiuGraphicInterface
+
+
+# ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ GODOT ░░░░
+func _ready() -> void:
+	super()
+	
+	Cursor.replace_frames($Cursor)
+	Cursor.show_cursor()
+	
+	$Cursor.hide()
+	
+	$Menu.hide()
+	
+	E.current_command = SierraCommands.Commands.WALK
+	
+	%SierraSettingsPopup.option_selected.connect(_on_settings_option_selected)
+
+
+func _input(event: InputEvent) -> void:
+	if D.current_dialog: return
+	
+	if event is InputEventMouseMotion:
```

Comment:
This logic should belong to a menu-tied script.

Reply:
Done. And thanks to this I decided to create a new Graphic Interface signal to open popups. Not 100% about this, but looks fine.